packages/orchestration/test/exos/portfolio-holder-kit.test.ts

```diff
@@ -24,7 +24,7 @@ test('portfolio holder kit behaviors', async t => {
     },
   });
 
-  const makeTestCOAKit = prepareMakeTestCOAKit(t, common, {
+  const makeTestCOAKit = prepareMakeTestCOAKit(t, common, undefined, {
```

Comment:
The `chore: correct parameters in call to prepareMakeTestCOAKit()` commit can be dropped because it makes this change too early